bindings/c/fdb_c.cpp

```diff
@@ -18,6 +18,8 @@
  * limitations under the License.
  */
 
+#include "fdbclient/BlobGranuleCommon.h"
```

Comment:
I would suggest introducing a new header and cpp for blobgranule-related functionality, e.g. fdb_c_blobranule.h & fdb_c_blobranule.cpp, because it is a quite specific functionality and reached certain level of complexity

Reply:
I agree, but given the complexity of this PR already, and its dependencies, let's address in a follow-up PR.